language-design.md

```diff
@@ -62,7 +175,50 @@ the commenters?
 
 **Response**
 
-
+Pavlus is arguing that programming languages should be designed to be easier
+for the average person to understand and learn. While we agree with this
+general argument of the article, the actual specifics Pavlus uses to bolster
+his claim are not convincing. The commenters point out that Pavlus is
```

Comment:
agreed, Pavlus got in way too deep and just didn't really know what he was talking about. he was trying to write a punchy piece but missed the subtleties that make it a tough conversation